### PR TITLE
[KARAF-2511] Add karaf-servlet-example

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

File: examples/karaf-servlet-example/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Abstract
+
+This example show how to register a servlet in the Karaf HTTP Service.
+
+It uses different approaches:
+
+1. Using `@WebServlet` annotation using the Karaf annotation scanner
+2. Using Karaf Servlet whiteboard pattern with blueprint
+3. Using Karaf Servlet whiteboard pattern with SCR
+
```
Comment (fpapon) [inline examples/karaf-servlet-example/README.md]: We can add "4. Using the Karaf HTTP service." for the karaf-servlet-example-registration ?

**Review by fpapon — CHANGES_REQUESTED** [review]
See my previous comment please

File: examples/karaf-servlet-example/README.md (hunk 1 above)
Comment (jbonofre) [reply examples/karaf-servlet-example/README.md]: Good catch, thanks !

**Comment by jbonofre** [comment]
@fpapon I've updated the README.md according to your comment. Can you please do a new review round ? Thanks !

**Review by fpapon — APPROVED** [review]

**Comment by jbonofre** [comment]
Squashed.

**Pull request merged**